vkubelet/vkubelet.go

```diff
@@ -161,8 +161,8 @@ func (s *Server) reconcile(ctx context.Context) {
 
 	var failedDeleteCount int64
 	for _, pod := range deletePods {
-		logger := logger.WithField("pod", pod.Name)
-		logger.Debug("Deleting pod '%s'\n", pod.Name)
```

Comment:
just about to fix the issue. can you also add the namespace?

Reply:
Fixed.